src/main/java/com/bandwidth/sqs/action/SetQueueAttributesAction.java

```diff
@@ -7,17 +7,17 @@
 import com.amazonaws.services.sqs.model.transform.SetQueueAttributesRequestMarshaller;
 import com.amazonaws.services.sqs.model.transform.SetQueueAttributesResultStaxUnmarshaller;
 import com.bandwidth.sqs.action.adapter.SqsAwsSdkAction;
-import com.bandwidth.sqs.queue.SqsQueueAttributes;
+import com.bandwidth.sqs.queue.SqsQueueAttributeChanges;
```

Comment:
Is the stuff in `SqsQueueAttributeChanges` "changes" or "changed attributes"?

Looking further down, there are params like `SqsQueueAttributeChanges attributes`. An instance is called "attributes" but is of type "changes". Would it be better to call the attrib "changes" or should the type be "attributes"?

`SqsQueueChangedAttributes attributes` vs `SqsQueueAttributeChanges changes`